assets/js/stripe-payment-request.js

```diff
@@ -639,33 +637,6 @@ jQuery( function( $ ) {
 					} );
 				});
 			} );
-
-			// Block the payment request button as soon as an "input" event is fired, to avoid sync issues
-			// when the customer clicks on the button before the debounced event is processed.
-			$( '.quantity' ).on( 'input', '.qty', function() {
-				$( document.body ).trigger( 'wc_stripe_block_payment_request_button' );
-			} );
-
-			$( '.quantity' ).on( 'input', '.qty', wc_stripe_payment_request.debounce( 250, function() {
-				$( document.body ).trigger( 'wc_stripe_block_payment_request_button' );
-				paymentRequestError = [];
-
-				$.when( wc_stripe_payment_request.getSelectedProductData() ).then( function ( response ) {
```

Comment:
Please read the third list item on the PR's description to know about why I did remove this.